Client/Frontend/Browser/TabDisplayManager.swift

```diff
@@ -35,9 +35,8 @@ class TabDisplayManager: NSObject {
     let tabReuseIdentifer: String
 
     var searchedTabs: [Tab] = []
-    var searchActive: Bool = false
+    var searchActive: Bool { return !searchedTabs.isEmpty }
```

Comment:
minor cleanup, change to calculated var.

Reply:
This won’t handle the case where the search result returns no tabs

Reply:
Thanks for checking. Maybe in future this could be a nil instead, I would like to see less state vars in here. There is a new utility function to get the most recent tab for a given mode, so maintaining previous tab can be removed also.